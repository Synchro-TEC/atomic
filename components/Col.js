import React from "react";
import PropTypes from "prop-types";
import cx from "classnames";

const Col = ({ className, style, withGutter, children, ...otherProps }) => {
  const baseClass = "sv-column";
  const rootClass = cx(baseClass, className);

  return (
    <div className={rootClass} style={style} {...otherProps}>
      {children}
    </div>
  );
};

Col.propTypes = {
  className: PropTypes.string,
  style: PropTypes.object,
};

export default Col;
