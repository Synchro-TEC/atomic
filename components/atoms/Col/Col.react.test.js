import React from 'react';
import Col from './Col';

describe('Col Component', () => {
  test('Render Col Component', () => {
    const col = shallow(<Col />);
    expect(col).toMatchSnapshot();
  });
});
