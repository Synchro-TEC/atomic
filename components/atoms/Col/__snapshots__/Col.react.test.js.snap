// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Col Component Render Col Component 1`] = `
<div
  className="sv-column"
/>
`;
